### ISSUE TITLE
Model evaluation

### ISSUE THREAD

**Issue by Yiyuan80** [miamianwei/821_final_project#4]
Track the loss and accuracy of the validation set to determine the best hyperparameters.
